core/validation/failures.py

```diff
@@ -386,22 +386,6 @@ def to_message(self) -> tuple[str | None, str | None, str]:
         return None, None, PRETRANSFORMATION_FAILURE_MESSAGE_BANK[self.value_descriptor]
 
 
-@dataclass
-class NoInputFailure(PreTransFormationFailure):
-    """Class representing a no input failure."""
-
-    value_descriptor: str
-
-    def __str__(self):
-        """
-        Method to get the string representation of the no input failure.
-        """
-        return f"No Input Failure: Expected an input value for {self.value_descriptor}"
-
-    def to_message(self) -> tuple[str | None, str | None, str]:
-        return None, None, PRETRANSFORMATION_FAILURE_MESSAGE_BANK[self.value_descriptor]
```

Comment:
Are there redundant values in this message bank we can also remove?

Reply:
No, the message for Place Name works both for WrongInput and NoInput, completely removing the need for the latter